conformance/tests/mesh-split.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2022 The Kubernetes Authors.
```

Comment:
A lot of these files have old copyright dates, are they just being copied from Istio codebase?

Reply:
copied from other files in this repo